apps/namadillo/src/workers/ShieldedSyncWorker.ts

```diff
@@ -0,0 +1,65 @@
+import init, { initMulticore } from "@heliax/namada-sdk/inline-init";
+
+import { getSdk } from "@heliax/namada-sdk/web";
+
+export type ShiededSyncPayload = {
+  vks: string[];
+  rpcUrl: string;
+  maspIndexerUrl?: string;
+};
+
+export type ShieldedSyncMulticore = {
+  type: "shielded-sync-multicore";
+  payload: ShiededSyncPayload;
+};
+
+export type ShieldedSyncSinglecore = {
+  type: "shielded-sync-singlecore";
+  payload: ShiededSyncPayload;
+};
+
+export type ShieldedSyncMessageType =
+  | ShieldedSyncMulticore
+  | ShieldedSyncSinglecore;
+
+self.onmessage = async (e: MessageEvent<ShieldedSyncMessageType>) => {
+  const { type, payload } = e.data;
+
+  switch (type) {
+    case "shielded-sync-singlecore": {
+      const { cryptoMemory } = await init();
+      //eslint-disable-next-line
+      console.log("Syncing with single core");
```

Comment:
Maybe we could change these to `console.info` to avoid the eslint-disable? That might work